### DISCUSSION TITLE
If your comment in a repository's issues includes a preformatted/inline code field, you can't easily see the far right side of it

### DISCUSSION THREAD

**Discussion by aki-k** [community/community#66723] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The screenshot is of this comment: 

https://github.com/microsoft/WSL/issues/9968#issuecomment-1712987361

![github-display-bug](https://github.com/community/community/assets/2235371/8e3b5b1c-2da8-45ae-bdb0-cb50b4ba2bb2)

**Comment by amineamdouni** [comment]
Use the horizontal scroll bar at the bottom of the code block to scroll to the right.
 Resize your browser window to make it wider, allowing you to see more of the code.
3Zoom out on your browser using Ctrl/Cmd + - to fit more content within the viewport.

**Comment by aki-k** [comment]
I'm reading the issue page on Android Opera. I can only zoom the page but I can't resize the window.  

Can't you fix this issue properly instead of offering me a kludge? 

One solution would be to not draw that double square (copy to clipboard) icon on top of the preformatted/inline code field.
